api/src/main/java/org/pmiops/workbench/db/model/DbWorkspace.java

```diff
@@ -797,6 +809,25 @@ public DbWorkspace setUsesTanagra(boolean usesTanagra) {
     return this;
   }
 
+  @Enumerated(EnumType.STRING)
+  @Column(name = "rp_aian_research_type")
+  public AIANResearchType getAianResearchType() {
+    return aianResearchType;
+  }
+
+  public void setAianResearchType(AIANResearchType aianResearchType) {
```

Comment:
The getters/setters seem a little strange here, because it uses Aian instead of AIAN.

If I rename it to the later, the mapping breaks. Happy to adjust if you have a preference.